### PR TITLE
feat(ca): fix logs, refactor formula

### PR DESCRIPTION
**Pull request by torgeadelin**
# COMPLETES https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-465734

## This pull request addresses

- refactor latency formula
- fix CA logs

## by making the following changes
- refactor

### Change Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [x] Internal code refactor

## The following scenarios where tested
- manual test, unit test
### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/@webex/internal-plugin-metrics/src/utils.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/prefer-default-export */
+/**
+ * Generates common metadata for errors
+ * @param {any} error
+ * @returns {object}
+ */
+export const generateCommonErrorMetadata = (error) => {
```
Comment (shnaaz) [inline packages/@webex/internal-plugin-metrics/src/utils.ts:7]: There should be a test for this

File: packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics-batcher.ts [hunk 2]
```diff
@@ -238,6 +258,50 @@ describe('plugin-metrics', () => {
...
+            ])
+            assert.deepEqual(loggerLogCalls[1].args, [
+              "call-diagnostic-events -> ",
+              "CallDiagnosticEventsBatcher: @submitHttpRequest#call-diagnostic-metrics-batch-9. Sending the request:",
+              `payload: [{"eventPayload":{"event":"my.event","origin":{"buildType":"test","networkType":"unknown"}},"type":["diagnostic-event"]}]`
+            ]);
+
+          })
+          .then(() => promise).catch((err) =>{
+            console.log(err)
```
Comment (shnaaz) [inline packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics-batcher.ts:293]: Should removed console log

File: packages/@webex/internal-plugin-metrics/src/utils.ts (hunk 1 above)
Comment (torgeadelin) [reply packages/@webex/internal-plugin-metrics/src/utils.ts:7]: added

File: packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics-batcher.ts (hunk 2 above)
Comment (torgeadelin) [reply packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics-batcher.ts]: removed

**Review by Coread — APPROVED** [review]

**Pull request merged**